addons/op-badge/addon.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Show OP badge in forums and projects",
+  "description": "Adds a OP badge to the user that created a forum topic or project.",
```

Comment:
Name and description need to be changed. Ideally at least one of them should mention "original poster" somewhere. Possibly "project author" too.
Should also mention the word "comment(s)".

Reply:
@DNin01 Help?